packages/marshal/src/encodeToCapData.js

```diff
@@ -117,7 +117,7 @@ export const makeEncodeToCapData = (encodeOptions = {}) => {
    * Readers must not care about this order anyway. We impose this requirement
    * mainly to reduce non-determinism exposed outside a vat.
    *
-   * @param {Passable} passable
+   * @param {any} passable
```

Comment:
Curious: Why did this "regress" from `Passable` to `any`?

(scare quotes because the old `Passable` meant `any`. So the real question is: Why couldn't this become the new `Passable`?)

Likewise for similar changes elsewhere.

Reply:
All the string replacements of old-Passable to `any` is because the new (non-any) Passable caused too many errors and I judged solving them to not be worth it. Usually because the following code relied heavily on dynamic type inspection and would not benefit from the static type information. This was mostly in `switch` statements where TS cannot statically infer the type narrowing being used.